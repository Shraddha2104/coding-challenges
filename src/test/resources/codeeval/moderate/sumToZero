-23,-15,66,-91,21,32,-70,17,17,-55,-51,19,-56,-37,59,45,6,58
22,34,81,-36,90,35,-89,-54,-89,-76,3,4,16,-94,-82,-14,-73,-51,4
-67,89,47,32,-67,-70,95,-74,18,-22,76,26,74,90,-69,2
94,40,44,10,-92,36,44,39,-14,46,-81,21,94,-47,13
56,-76,31,-30,-90,65,17,50,-80,-97,70,-58,-100
4,65,-5,-97,-37,-22,67,44,32,19,39,-39,32,1,-32,96,-38,4
-11,14,24,-64,19,19,-38,70,-50,-31,52,89,-30,-32,39
97,100,51,-73,-23,89,-64,-50,-6,4,-68,69,-37,31,-61,-89
42,-19,69,72,-27,26,-15,35,-84,9,66,91,-41,66,-87,-41,-55
95,-91,47,-1,37,1,86,-87,64,91,-14,66,27
-100,-43,100,92,-97,-11,28,79,92,-40,84,-19,27
-90,-59,-79,24,27,-92,-81,28,-79
-29,99,-62,-9,-65,6,41,-36,-4,-87,-99
87,-81,-82,88,6,51,-5,-74,-97,-17,5,10,47,-82,91,55,-57
-2,43,70,-85,-60,-38,-39,-93,-42,-20,-18,-21,17,64
-28,-48,41,20,-29,-74,54,4,97,58,-21,-27,100,57
-20,16,46,-2,-53,32,44,-40,53,-78,65,88,-76,5
-9,8,84,66,-26,87,99,74,-99,18,-70,75,15,-19
-40,90,59,72,-60,-31,36,81,86,-45,-15,-87,12,84,-63,-35
-100,26,20,-14,-31,-7,-87,-38,-25,-61,16,-64,43,-7,11,84,-34,-23,-84
2,3,1,0,-4,-1
0,-1,3,-2
