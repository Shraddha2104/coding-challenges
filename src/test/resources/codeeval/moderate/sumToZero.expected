16
17
9
3
2
16
4
3
4
2
3
2
0
10
8
4
5
2
9
8
2
1
